templates/CRM/Contact/Form/Task/SaveSearch.tpl

```diff
@@ -25,16 +25,22 @@
 *}
 <div class="crm-form-block crm-block crm-contact-task-createsmartgroup-form-block">
   <h3>{ts}Smart Group{/ts}</h3>
-  {if $qill[0]}
+  <div class="help">
+    <p>{ts}This smart group will stay up-to-date with all contacts who meet the search criteria.{/ts}</p>
+    {if !empty($partiallySelected)}
+      <p>{ts}NOTE: Contacts selected on the search results are not relevant here; all contacts that meet the following criteria will be in the group.{/ts}</p>
```

Comment:
I think no, because this message is specifically about /creating/ a new 
smart group. The reason for the message is to be clear about what the 
user is doing in this moment (which is taking search results and saving 
a smart group) thus the note clarifying why any checkboxes they may have 
ticked in the search results is irrelevant.

As for what they might do with the smart group in the future, there's a 
lot to say about that and I think the link to the book is better than 
adding random tips here.

On 07/02/2015 01:43 AM, seamuslee001 wrote:
>
> In templates/CRM/Contact/Form/Task/SaveSearch.tpl 
> <https://github.com/civicrm/civicrm-core/pull/6111#discussion_r33748650>:
>
> > @@ -25,16 +25,22 @@
> >  *}
> >  <div class="crm-form-block crm-block crm-contact-task-createsmartgroup-form-block">
> >    <h3>{ts}Smart Group{/ts}</h3>
> > -  {if $qill[0]}
> > +  <div class="help">
> > +    <p>{ts}This smart group will stay up-to-date with all contacts who meet the search criteria.{/ts}</p>
> > +    {if !empty($partiallySelected)}
> > +      <p>{ts}NOTE: Contacts selected on the search results are not relevant here; all contacts that meet the following criteria will be in the group.{/ts}</p>
>
> @colemanw <https://github.com/colemanw> should there be a note about 
> if people unsubscribe from the smart group that they won't appear in 
> the group?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/civicrm/civicrm-core/pull/6111/files#r33748650>.
>